### PR TITLE
LPD-28622 Fix DeleteAPIPropertiesToAPIPropertiesUpgradeProcess

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPD-28622

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:test:upgrade

**Comment by 4lejandrito** [comment]
ci:stop

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward:force

File: modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+		}
+
+		Assert.isNull(
+			_objectRelationshipLocalService.
+				fetchObjectRelationshipByExternalReferenceCode(
+					"L_API_PROPERTIES_TO_API_PROPERTIES",
+					objectDefinition.getObjectDefinitionId()));
+	}
+
+	private static StopWatch _originalStopWatch;
```
Comment (ccorreagg) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java:117]: I would rename it to `_stopWatch` as there are no more stopWatch

**Review by ccorreagg — COMMENTED** [review]
See my comments

File: modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+	@Inject(
+		filter = "(&(component.name=com.liferay.headless.builder.internal.upgrade.registry.HeadlessBuilderUpgradeStepRegistrator))"
```
Comment (ccorreagg) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java:120]: I think it is not necessary the umpersand as you only have one condition

Comment (ccorreagg) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java:120]: Even it is possible that no parenthesis is needed

File: modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java:117]: I based it on https://github.com/liferay/liferay-portal/blob/dd44186cbc815ece23a21f8402470fbaafebeb33/modules/apps/portal/portal-upgrade-test/src/testIntegration/java/com/liferay/portal/upgrade/test/UpgradeRecorderTest.java#L452.

File: modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java (hunk 2 above)
Comment (4lejandrito) [reply modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/upgrade/v0_1_0/test/DeleteAPIPropertiesToAPIPropertiesUpgradeProcessTest.java]: See 371bd64c94c4b011aee83b477762915523cfff3e.

**Comment by 4lejandrito** [comment]
ci:forward:force

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**